src/lightning/data/streaming/item_loader.py

```diff
@@ -67,12 +67,15 @@ def generate_intervals(self) -> List[Tuple[int, int]]:
     def load_item_from_chunk(self, index: int, chunk_index: int, chunk_filepath: str, begin: int) -> bytes:
         offset = (1 + (index - begin) if index >= begin else index + 1) * 4
 
+        if chunk_filepath in self._chunk_filepaths and not os.path.isfile(chunk_filepath):
+            del self._chunk_filepaths[chunk_filepath]
+
         if chunk_filepath not in self._chunk_filepaths:
             while not os.path.exists(chunk_filepath):
-                sleep(0.001)
+                sleep(0.01)
```

Comment:
Maybe move it (from this and other similar occurrences) to a constant in the future?

Reply:
Yes, will do ;)